voice-recognition

```diff
@@ -0,0 +1,103 @@
+from numpy import *
+from scipy import *
+import sys, getopt
+import numpy as np
+import scipy.io.wavfile
+from scipy import signal as sig
+import glob
+
+
+
+
+def main(argv):
+    # czytamy plik i wyciagamy czestotliwosc probkowania
```

Comment:
Comments which describe code- it will be better tu put these into the function with descriptive name